package io.pivotal.trilogy.testrunner

class InputArgumentException(message: String, cause: Throwable): RuntimeException(message, cause)